files/codemirror.spell-checker/update.json

```diff
@@ -3,6 +3,7 @@
   "name": "codemirror-spell-checker",
   "repo": "NextStepWebs/codemirror-spell-checker",
   "files": {
+    "basePath": "dist",
     "include": ["spell-checker.min.js", "spell-checker.min.css", "en_US.aff", "en_US.dic"]
```

Comment:
There are no such files in the repo https://github.com/jsdelivr/jsdelivr/pull/6460